internal/cli/atlas/accesslogs/list.go

```diff
@@ -119,6 +119,8 @@ func ListBuilder() *cobra.Command {
 		Short:   "Retrieve the access logs of a cluster identified by the cluster's name or hostname.",
 		Long:    fmt.Sprintf(usage.RequiredRole, "Project Monitoring Admin"),
 		Args:    require.NoArgs,
+		Example: fmt.Sprintf(`  # Return a JSON-formatted list of all access list entries for the project with ID 618d48e05277a606ed2496fe for the cluster named Cluster0:		
```

Comment:
Change: This isn't quite right. These aren't access list entries, they are a list of authentication requests made against the cluster. See: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Access-Tracking/operation/listAccessLogsByClusterName